### PR TITLE
Add new: aurora-console

### PR DESCRIPTION
**Pull request by xuri**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

**Comment by Luxurioust** [comment]
aurora-concole is a web-based Beanstalk queue server console. I have submit [one month ago](https://github.com/caskroom/homebrew-cask/pull/25544), and I also try submit to [Homebrew](https://github.com/Homebrew/homebrew-core/pull/6751), but they don’t accept it, so I come back again. please help me to add this app to cask, thanks a lot.

File: Casks/aurora-console.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'aurora-console' do
+  version '1.4'
+  sha256 '44d3de24bf6e3391b165effb039d5cca75fa428ec6d6cb1f6bddd7ad3bca0319'
+
+  url "https://github.com/Luxurioust/aurora/releases/download/#{version}/aurora_darwin_amd64_v#{version}.tar.gz"
```
Comment (vitorgalvao) [inline Casks/aurora-console.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/aurora-console.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  name 'aurora'
+  homepage 'https://github.com/Luxurioust/aurora/'
+
+  license :mit
+
+  binary 'aurora', target: 'aurora'
```
Comment (vitorgalvao) [inline Casks/aurora-console.rb]: Remove `, target: 'aurora'`.

File: Casks/aurora-console.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'aurora-console' do
```
Comment (vitorgalvao) [inline Casks/aurora-console.rb]: `luxurioust-aurora`

File: Casks/aurora-console.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  version '1.4'
+  sha256 '44d3de24bf6e3391b165effb039d5cca75fa428ec6d6cb1f6bddd7ad3bca0319'
+
+  url "https://github.com/Luxurioust/aurora/releases/download/#{version}/aurora_darwin_amd64_v#{version}.tar.gz"
+  name 'aurora'
+  homepage 'https://github.com/Luxurioust/aurora/'
```
Comment (vitorgalvao) [inline Casks/aurora-console.rb]: `https://xuri.me/aurora/`

File: Casks/aurora-console.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/aurora-console.rb]: This will need a comment after you change the `homepage`. Run the `style` command again for more information.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/aurora-console.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+  license :mit
```
Comment (vitorgalvao) [inline Casks/aurora-console.rb]: Remove `license`, it has been deprecated and is no longer in use.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
In the spirit of deduplication, I’ll refuse this PR for now.

Homebrew did not refuse your package outright, but instead refused it *for now*. I’d rather not have it here and later duplicated there. Please wait a bit longer and then resubmit (or ask another contributor to, since they’re less likely to refuse if you’re not the one submitting your own work).

In the meantime, having your own tap is the best solution.

Thank you for the interest and contribution.

**Pull request closed without merging**